hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -1058,14 +1059,14 @@ private boolean match(int expected) {
         @SuppressWarnings("unused")
         private void dump() {
             this.seek();
-            System.out.println(">>>> scan from " + this.table + ": "  +
-                               (this.keyBegin == null ? "*" :
-                                StringEncoding.format(this.keyBegin)) +
-                               (this.iter.isValid() ? "" : " - No data"));
+            LOG.info(">>>> scan from " + this.table + ": "  +
+                    (this.keyBegin == null ? "*" :
+                            StringEncoding.format(this.keyBegin)) +
+                    (this.iter.isValid() ? "" : " - No data"));
             for (; this.iter.isValid(); this.iter.next()) {
-                System.out.println(String.format("%s=%s",
-                                   StringEncoding.format(this.iter.key()),
-                                   StringEncoding.format(this.iter.value())));
+                LOG.info(String.format("%s=%s",
```

Comment:
Would you remove "String.format"

such as 

https://github.com/apache/incubator-hugegraph/pull/1838#discussion_r855050621